backend/pkg/log/server/log_service_server_impl.go

```diff
@@ -58,15 +59,13 @@ func (lss *LogServiceServerImpl) Export(
 						lss.logger.Error("Failed to parse log with message", zap.Error(err))
 						return
 					}
-					err = lss.cache.Put(processCtx, logWithClusterId.ClusterId, []model.LogEntry{logWithClusterId})
+					insertCtx, insertCancel := context.WithTimeout(context.Background(), 10*time.Second)
+					defer insertCancel()
+					err = lss.cache.Put(insertCtx, logWithClusterId.ClusterId, []model.LogEntry{logWithClusterId})
 					if err != nil {
 						lss.logger.Error("Failed to put log in cache", zap.Error(err))
 						return
 					}
-					err = lss.countService.CountAndUpdateOccurrences(processCtx, logWithClusterId, buckets)
```

Comment:
Removed count, will defer it to a cronjob so that the consumption of data is separated from its analysis